src/assets/scripts/base.js

```diff
@@ -171,9 +190,23 @@ function renderSerpEntry(index, hit) {
   } else {
     d.append($("<h4></h4>").html((index + 1) + ". " + hit._source.title));
   }
+
   if (typeof(hit._source.breadcrumb) !== "undefined" && hit._source.breadcrumb.length > 0) {
-    d.append($("<p class='sbreadcrumb'></p>").html("/" + hit._source.breadcrumb.join("/") + "/"));
+    var top = hit._source.breadcrumb[0].toLowerCase();
+    if (top === 'blog') {
+      var breadcrumb = 'Blog post'
+      if (typeof(hit._source.published) !== "undefined") {
```

Comment:
Did this work? IMO the syntax to check for undefined is like this:

```suggestion
      if (typeof hit._source.published !== "undefined") {
```